module/applications/global/prompt-factory.mjs

```diff
@@ -107,6 +107,16 @@ export default class PromptFactory {
     };
   }
 
+  static get chooseNoDisciplineButton() {
```

Comment:
Splitting hairs here but :D I think `noDisciplineButton` should be enough, since it could be used in other contexts other than choosing something. The choosing is not part of the Button itself, it just says "noDiscipline"...